devex/vsextension/vsix-release-process.md

```diff
@@ -0,0 +1,78 @@
+# VS Extension Release Process
+
+The process used to build and release the nuget package from these
+sources is as follows.  Currently the signature on the release is
+a Microsoft signature generated using the Microsoft [PODSS](https://aka.ms/podss) process.
+In the future, this process will change to a use a non-Microsoft
+signature once another signing process is defined for Open Enclave.
+
+1. Build the latest [Cross-Platform Open Enclave SDK NuGet Package](https://github.com/HernanGatta/openenclave-cross-plat-nuget).
```

Comment:
This can point to `devex/cross-nuget` once #3638 is merged.

Reply:
> This can point to `devex/cross-nuget` once #3638 is merged.

Done